pubtools/_quay/tag_images.py

```diff
@@ -199,26 +251,42 @@ def tag_images(
         quay_password,
         remote_exec,
         ssh_remote_host,
+        container_exec,
+        container_image,
         send_umb_msg,
         umb_urls,
         umb_cert,
     )
 
     if remote_exec:
         accept_host = not ssh_reject_unknown_host if ssh_reject_unknown_host else True
-        executor = RemoteExecutor(
+        executor_class = functools.partial(
+            RemoteExecutor,
             ssh_remote_host,
             ssh_username,
             ssh_key_filename,
             ssh_password,
             ssh_remote_host_port,
             accept_host,
         )
+    elif container_exec:
+        if isinstance(docker_timeout, str):
+            docker_timeout = int(docker_timeout)
+        executor_class = functools.partial(
+            ContainerExecutor,
+            container_image,
+            docker_url,
+            docker_timeout,
+            docker_verify_tls,
+            docker_cert_path,
+        )
     else:
-        executor = LocalExecutor()
+        executor_class = functools.partial(LocalExecutor)
```

Comment:
Isn't this a longer way to say `executor_class = LocalExecutor`? :)

Reply:
:D . But I'd prefer to keep it this way because maybe LocalExecutor will accept some constructor arguments in the future. Also, I feel the code is a bit less convoluted when all executors are created the same way. Is it okay?